crypto/asn1/a_d2i_fp.c

```diff
@@ -62,6 +62,8 @@ void *ASN1_item_d2i_bio(const ASN1_ITEM *it, BIO *in, void *x)
     void *ret = NULL;
     int len;
 
+    if (in == NULL)
+        return NULL;
```

Comment:
Not sure about this bit. I've found it annoying with other functions that sometimes produce an error report on error and sometimes not.

Reply:
Indeed the situation whether functions produce an error on NULL input is pretty inconsistent.
Apparently most functions don't.

Reply:
In this case IMO it is best not to produce an error on NULL input because, as also @t8m commented, the actual error in this case was already elsewhere, and re-reporting it here would just obscure the real place/reason.